charts/caraml/ci/ci-values.yaml

```diff
@@ -0,0 +1,181 @@
+mlp:
+  deployment:
+    replicaCount: "1"
+    resources:
+      requests:
+        cpu: 100m
+        memory: 256Mi
+      limits:
+        cpu: "250m"
+        memory: 512Mi
+
+merlin:
```

Comment:
Do we need to disable mlp under merlin?
```
merlin:
  mlp:
     enabled: false
```

Reply:
It is disabled under the main values file for caraml. Since when installing caraml the default would be the same, mlp deployed via caraml and the mlp under merlin is disabled.

Reply:
check https://github.com/caraml-dev/helm-charts/pull/33/files#diff-5ea3bd7a60d2abe2dc8ff60efc8f412a9711b45b0fa9e8f9091845da7fdb92d9R32

Reply:
got it